_docs/concepts/clusters.md

```diff
@@ -0,0 +1,21 @@
+---
+layout: guide
+title: StorageOS Docs - Clusters
+anchor: concepts
+module: concepts/clusters
+---
+
+# Clusters
+
+StorageOS clusters represent groups of nodes which run a common distributed
+control plane, and aggregate their storage into one or more [pools]({% link
+_docs/concepts/pools.md %}).
+
+Typically, a StorageOS cluster maps one-to-one to a Kubernetes (or similar
+orchestrator) cluster, and we expect our container to run on all of the nodes
+within that cluster.
```

Comment:
"we expect our container to run on all of the nodes
within that cluster. "

--> Probably not in the master nodes unless they host workloads.